mx4-core.h

```diff
@@ -109,8 +110,8 @@ struct mx4_spi_device
 };
 
 #define MX4_IO_DRV_NAME	"mx4_io_spi"
-#define DRIVER_VERSION	"2.3"
-
+#define DRIVER_VERSION	"2.4"
+//#define DEBUG "1"
```

Comment:
Good, but please describe in the commit message why these two lines were changed.